packages/calling/src/CallHistory/constants.ts

```diff
@@ -11,3 +11,9 @@ export const SET_READ_STATE_SUCCESS_MESSAGE = 'Missed calls are read by the user
 export const SUCCESS_MESSAGE = 'SUCCESS';
 export const STATUS_CODE = 'statusCode';
 export const USER_SESSIONS = 'userSessions';
+export const V1 = 'v1';
+export const UC = 'uc';
+export const CONFIG = 'config';
```

Comment:
Please stick to the format of the constants file by sorting it in alphabetical order.
We can expand the V1 to VERSION_1 = 'v1' and UC to UNIFIED_COMMUNICATIONS = 'uc'.

Reply:
Thank you for the feedback. I will take care of it.

Reply:
I have updated the constants file to sort the entries in alphabetical order. 

Thank you!